string node_name
string description